src/test/resources/test-application.properties

```diff
@@ -1,26 +1,34 @@
-configuration.cache.expire=120
-vendor.id.cache.expire=600
-vendor.cache.expire=86400
-package.cache.expire=86400
-resource.cache.expire=86400
-title.cache.expire=86400
-configuration.base.url.default=https://sandbox.ebsco.io
-holdings.status.check.delay=50
-holdings.status.retry.count=5
+access.types.default.number.limit.value=3
+
+db.exception.translator.name=postgresql
+
+# Possible values are TransactionLoadServiceFacade, DefaultLoadServiceFacade
+# Qualifier of a bean that implements LoadServiceFacade interface.
+holdings.load.implementation.qualifier=DefaultLoadServiceFacade
+holdings.load.retry.count=2
+holdings.load.retry.delay=1
+holdings.page.retry.count=2
+holdings.page.retry.delay=1
 holdings.report.status.check.delay=120000
 holdings.report.status.retry.count=30
-holdings.status.audit.expiration.period=2592000000
+holdings.snapshot.refresh.period=60000
 holdings.snapshot.retry.count=2
 holdings.snapshot.retry.delay=1
-holdings.snapshot.refresh.period=60000
-holdings.page.retry.count=2
-holdings.page.retry.delay=1
-holdings.load.retry.delay=1
-holdings.load.retry.count=2
+holdings.status.audit.expiration.period=2592000000
+holdings.status.check.delay=50
+holdings.status.retry.count=5
 holdings.timeout=60000
-# Qualifier of a bean that implements LoadServiceFacade interface.
-# Possible values are TransactionLoadServiceFacade, DefaultLoadServiceFacade
-holdings.load.implementation.qualifier=DefaultLoadServiceFacade
-access.types.default.number.limit.value=3
-db.exception.translator.name=postgresql
+
+# Credentials properties
+kb.ebsco.credentials.name.length.max=255
+kb.ebsco.credentials.url.default=https://sandbox.ebsco.io
```

Comment:
Really minor comment -- could this default to https://api.ebsco.com? This will be the default/used most of the time

Reply:
Yes, I will change it.

Reply:
Changed